rero_ils/modules/ext.py

```diff
@@ -149,6 +150,95 @@ def load_actions(sender, app):
         access_ext.register_action(obj_or_import_string(action))
 
 
+class _REROILSAppState(object):
+    """RERO-ILS App state class.
+
+    # TODO: we can also migrate other method like:
+            - rero_ils.utils:
+                * get_endpoint_configuration
+                * extracted_data_from_ref
+                * get_pid_type_from_schema
+                * get_schema_for_resource
+                * get_ref_for_pid
+        maybe we use this implementation everywhere is it possible.
+    """
+
```

Comment:
As discuss in real life, i think it's maybe overkill... but a nice way.
What is your opinion @jma ?

Reply:
If I understand well the purpose of this, it seems to me a bit overkill. We will not "feel" the difference in term of the use of the system. Moreover, this can have some side effects, for example in the test we modify the app.config on the fly and if we cache some informations, we need to invalidate the cache which can takes several hours to understand or discover. If we decide to do it anyway, we need to have a very good documentation.